java/core/src/main/java/com/google/protobuf/ByteString.java

```diff
@@ -279,8 +316,8 @@ public int compare(ByteString former, ByteString latter) {
       };
 
   /**
-   * Returns a {@link Comparator} which compares {@link ByteString}-s lexicographically
-   * as sequences of unsigned byte values between 0 and 255, inclusive.
+   * Returns a {@link Comparator} which compares {@link ByteString}-s lexicographically as sequences
+   * of unsigned bytes (i.e. values between 0 and 255, inclusive).
```

Comment:
Technically I think we want to keep the version without the parentheses. The internal version redid the line wrapping but the external version actually changed the wording slightly. (I only know this because I have been trying to do a sync to internal and have been staring at the same merge conflict.)